components/admin/requests/modals/EditPermitHolderInformationModal.tsx

```diff
@@ -62,6 +62,7 @@ export default function EditPermitHolderInformationModal({
             </ModalHeader>
             <ModalBody paddingY="20px" paddingX="4px">
               <PermitHolderInformationForm
+                type="replacement"
```

Comment:
Shouldn't we pass down the request type as a prop through this Modal component, rather than hard-coding it to `replacement`?

Reply:
Yep good point, fixed now!